kafka-client/src/main/java/dev/responsive/kafka/internal/db/mongo/SessionDoc.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2024 Responsive Computing, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dev.responsive.kafka.internal.db.mongo;
+
+import com.mongodb.BasicDBObject;
+import dev.responsive.kafka.internal.utils.SessionKey;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.Objects;
+import org.bson.codecs.pojo.annotations.BsonCreator;
+import org.bson.codecs.pojo.annotations.BsonProperty;
+
+public class SessionDoc {
+
+  public static final String ID = "_id";
+  public static final String VALUE = "value";
+  public static final String EPOCH = "epoch";
+  public static final String TOMBSTONE_TS = "tombstoneTs";
+
+  // Subfields of the composite key _id
+  public static final String ID_RECORD_KEY = "key";
+  public static final String ID_SESSION_END_TS = "sessionEndTs";
```

Comment:
nit: can we change the suffix from TS to MS? I know I myself used TS in the WindowDoc (and we should change that too) but it's pretty clearly a timestamp whereas MS at least tells us the units. And we tend to use `Ms` suffixes for similar variable names in Kafka Streams